pyramid_swagger/tween.py

```diff
@@ -542,10 +542,10 @@ def validate_response(response, validator_map):
 
 
 def prepare_body(response):
-    # content_type and charset must both be set to access response.text
-    if response.content_type is None or response.charset is None:
+    # content_type must be set to access response.text
+    if not response.content_type:
```

Comment:
What errors would you expect? I ran the test suite with `webob` 1.6.3 and it works just fine (provided you downgrade `pyramid` as well).